src/RamIndexAnalysis.h

```diff
@@ -346,6 +348,22 @@ class MinIndexSelection {
         }
         orders.push_back(std::move(totalOrder));
     }
+    
+    /** @Brief return the attribute position for each inequality that should be discharged */
+    // NOTE: For now, all inequalities will be discharged but later the lex-orders will be inspected
+    // If an inequality is not in the last position of a lex-order only then is it discharged
+    AttributeSet getAttributesToDischarge() {
+        AttributeSet attributesToDischarge;
+        for (auto search : searches) {
+            size_t arity = search.arity();
+            for (size_t i=0; i<arity; ++i) {
```

Comment:
Since this is passing, the style check must be broken.
```suggestion
            for (size_t i = 0; i < arity; ++i) {
```